applications/GeoMechanicsApplication/tests/test_element_lab.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsLabElementTests(KratosUnittest.TestCase):
+    """
+    This class contains some element tests, such as triaxial and oedoemter tests
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_triaxial_comp_6n(self):
```

Comment:
Can you describe what the test does in the docstring?

Reply:
e.g. tests mohrcoulomb model on 2d/3d column/block with lineload/pointload on top/bottom. With free side boundaries.

Reply:
I added a description for the test